airbyte-integrations/connectors/source-bing-ads/source_bing_ads/base_streams.py

```diff
@@ -206,13 +206,21 @@ def request_params(
             "ReturnAdditionalFields": self.additional_fields,
         }
 
+    def _transform_tax_fields(self, record: Mapping[str, Any]) -> Mapping[str, Any]:
+        tax_certificates = record["TaxCertificate"].get("TaxCertificates", {}) if record.get("TaxCertificate") is not None else {}
```

Comment:
Could we use 
`tax_certificates = record.get("TaxCertificate", {}).get("TaxCertificates")`
instead here?

Reply:
all cases covered [here](https://github.com/airbytehq/airbyte/pull/35891/files#diff-c3e75a3a7dbedfb989273b30919ff8c79eb6d0caa5ddd3faa7eb795ec9405bf8R14). if initial TaxCertificate value is null (None), get method returns None, instead of empty dict and we have AttributeError.

Reply:
Got it, thanks!